README.md

```diff
@@ -29,14 +29,14 @@ This project is led by an experienced rails developer, but I'm actively looking
 - [Deploy the app on Heroku](#deploy-the-app-on-heroku)
 - [Deploy on your own server](#deploy-on-your-own-server)
 - [Configure optional features](#configure-optional-features)
-  - [Give assistant access to your Google apps](#configuring-google-tools)
+  - [Configuring Google Tools](#configuring-google-tools)
```

Comment:
Thanks for catching the broken anchors below. But on this change, since our readme has gotten so long, I like the more descriptive "Give assistant access to your Google apps" on this one. It gives people a vague idea of what's possible whereas I don't the more succinct "Configuring Google Tools" hints at enough.

Reply:
As always, sorry if my editor's markdown formatter/linter "fixes" things in a way that aren't to be fixed. I'll try to keep any eye out for any regressions in future PRs.